management/server/http/users.go

```diff
@@ -31,33 +29,34 @@ func NewUserHandler(accountManager server.AccountManager, authAudience string) *
 // UpdateUser is a PUT requests to update User data
 func (h *UserHandler) UpdateUser(w http.ResponseWriter, r *http.Request) {
 	if r.Method != http.MethodPut {
-		http.Error(w, "", http.StatusBadRequest)
+		util.WriteError(status.Errorf(status.InvalidArgument, "wrong HTTP method"), w)
```

Comment:
this case should never happen because of mux handler assignment. But in any case, we can return 405 here.